doc/cpptraj.lyx

```diff
@@ -10388,7 +10388,7 @@ The following topology related commands are available:
 \begin_layout Standard
 \align center
 \begin_inset Tabular
-<lyxtabular version="3" rows="18" columns="2">
+<lyxtabular version="3" rows="21" columns="2">
```

Comment:
What we can do (and what we do with the Amber repository at the moment) is we could add a stage to build the manual inside Jenkins and then archive that manual so it can be easily downloaded.

That way we can add a link to it in the README file and not have to add it to git (which is bad at tracking often-changing binary files like PDFs).

Reply:
I love this idea - great thinking! Let's do it in a separate PR.